### PR TITLE
fix checkstyle for tired-storage, websocket and bc modules

### PR DESCRIPTION
**Pull request by nlu90**
### Modifications

fixed module `tiered-storage`, `pulsar-websocket` and `bouncycastle`

### Verifying this change
```
➜  sn-pulsar: mvn checkstyle:check -Dpulsar.basedir=.
[INFO] Apache Pulsar :: Tiered Storage :: Parent .......... SUCCESS [  0.048 s]
[INFO] Apache Pulsar :: Tiered Storage :: JCloud .......... SUCCESS [  0.395 s]
[INFO] Apache Pulsar :: Tiered Storage :: File System ..... SUCCESS [  0.058 s]
...
[INFO] Apache Pulsar :: Bouncy Castle :: Parent ........... SUCCESS [  0.033 s]
[INFO] Apache Pulsar :: Bouncy Castle :: BC ............... SUCCESS [  0.037 s]
[INFO] pulsar-client-messagecrypto-bc ..................... SUCCESS [  0.050 s]
...
```

```
➜  sn-pulsar git:(neng/pulsar-module-checkstyle) ✗ mvn checkstyle:check -Dpulsar.basedir=. -pl pulsar-websocket
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: osx
[INFO] os.detected.arch: x86_64
[INFO] os.detected.classifier: osx-x86_64
[INFO]
[INFO] -----------------< org.apache.pulsar:pulsar-websocket >-----------------
[INFO] Building Pulsar WebSocket 2.10.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-checkstyle-plugin:3.1.2:check (default-cli) @ pulsar-websocket ---
[INFO] You have 0 Checkstyle violations.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.048 s
[INFO] Finished at: 2022-01-13T18:04:04-08:00
[INFO] ------------------------------------------------------------------------
```

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [X] `no-need-doc` 
  
  checkstyle fix
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java [hunk 1]
```diff
@@ -18,27 +18,21 @@
  */
 package org.apache.pulsar.websocket.stats;
 
+import static io.netty.util.internal.PlatformDependent.maxDirectMemory;
```
Comment (nlu90) [inline pulsar-websocket/src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java:21]: This import is illegal based on the rules:
```
```
➜  sn-pulsar git:(neng/pulsar-module-checkstyle) mvn checkstyle:check -Dpulsar.basedir=. -pl pulsar-websocket
...
[ERROR] src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java:[21,1] (imports) IllegalImport: Illegal import - io.netty.util.internal.PlatformDependent.maxDirectMemory.
```

But I don't have a good idea on how to fix this one. Need some help on it.

Comment (nicoloboschi) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java:21]: https://github.com/apache/pulsar/blob/f25b4a520bb82c6a09d373c9d314ca8e55d1f57b/buildtools/src/main/resources/pulsar/checkstyle.xml#L135-L137

we can edit the rule in my opinion

Comment (Shoothzj) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/stats/JvmMetrics.java:21]: @nlu90 We can make a suppression on `PlatformDependent.maxDirectMemory`, it's useful and widely use, like
```java
// CHECKSTYLE.OFF: IllegalImport
import io.netty.util.internal.PlatformDependent;
// CHECKSTYLE.ON: IllegalImport
```

**Review by Shoothzj — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@nlu90 Please help resolve the conflicts

**Comment by nlu90** [comment]
@codelipenghui Please take a look

**Review by eolivelli — APPROVED** [review]
Lgtm

**Pull request merged**